README.md

```diff
@@ -1,10 +1,10 @@
-# Mobile menu
+# details popup
```

Comment:
- [ ] Please as the last reviewer said the popup should be responsive for all screen sizes upper than 370px there is a weird overflow on your popup at 375 please fix it and make sure it matches the design at that size.
![image](https://user-images.githubusercontent.com/74744787/232260931-69513527-a7bc-4ee5-8832-13e6ee873b97.png)